# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ScriptKind.h
# Commit: 0527d227c6b6
# Full Hash: 0527d227c6b6f3d6648704a01bfd1c090b971ea7
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 21:52:29
# Regressor Bug: 1742437
# File Overlap Count: 2
# Description:
#   Bug 1742437 - Move module loader files into a central folder; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137532
# ==============================================================================

diff -r 967e68ed5f56 -r 0527d227c6b6 js/loader/ScriptKind.h
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/loader/ScriptKind.h	Fri Feb 18 12:45:52 2022 +0000
@@ -0,0 +1,18 @@
+/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
+/* vim: set ts=8 sts=2 et sw=2 tw=80: */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef mozilla_dom_ScriptKind_h
+#define mozilla_dom_ScriptKind_h
+
+namespace mozilla {
+namespace dom {
+
+enum class ScriptKind { eClassic, eModule, eEvent };
+
+}  // namespace dom
+}  // namespace mozilla
+
+#endif